packages/core/src/server/index.ts

```diff
@@ -50,9 +50,12 @@ const formatError: FormatErrorHandler = (err) => {
 }
 
 function loadGeneratedSchema(): TypeSource {
-  return loadFilesSync(path.join(process.cwd(), '@generated'), {
-    extensions: ['graphql'],
-  })
+  return loadFilesSync(
+    path.join(process.cwd(), '@generated', 'schema.graphql'),
```

Comment:
question: why did you add 'schema.graphql' here?

Reply:
Undid this :) Was by mistake but there was no behavior change.